hardhat.config.ts

```diff
@@ -95,7 +95,7 @@ const config: HardhatUserConfig = {
     deployer: 4,
     governor: {
       // Gnosis multisigs
-      default: '0x1a00e1E311009E56e3b0B9Ed6F86f5Ce128a1C01',
+      hardhat: '0x1a00e1E311009E56e3b0B9Ed6F86f5Ce128a1C01',
```

Comment:
I'm changing this so that if we try to deploy to a new network, it fails instead of using the default address